openapi.yaml

```diff
@@ -15,34 +15,34 @@ security:
   - BearerAuth: [ ]
 
 tags:
-  - name: Make
+  - name: Listing Search
 
 paths:
-  /v1/makes:
-    get:
-      operationId: GetMakes
-      description: Returns a list of makes sorted alphabetically.
+  /public/v1/listings/facets:
+    post:
       tags:
-        - Make
-      parameters:
-        - $ref: "#/components/parameters/pageQueryParam"
-        - $ref: "#/components/parameters/sizeQueryParam"
+        - Listing Search
+      description: Retrieve facets based on specified query parameters.
+      operationId: GetFacets
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ListingFacetsDto'
+        required: true
       responses:
-        '200':
-          description: Makes returned.
+        "200":
+          description: Facets successfully returned.
           content:
-            application/json:
+            '*/*':
```

Comment:
Should we use " instead of '? And what's the reason for being more unspecific with */* instead of specific?